apps/web/public/static/locales/en/common.json

```diff
@@ -981,7 +981,7 @@
   "verify_wallet": "Verify Wallet",
   "create_events_on": "Create events on",
   "enterprise_license": "This is an enterprise feature",
-  "enterprise_license_description": "To enable this feature, have an administrator go to <2>/auth/setup</2> to enter a license key. If a license key is already in place, please contact <5>{{SUPPORT_MAIL_ADDRESS}}</5> for help.",
+  "enterprise_license_description": "To obtain a license for this feature, visit https://cal.com/sales. If a license key is already in place, please contact support@cal.com for help.",
```

Comment:
This won't work with 

<img width="780" alt="Screenshot 2024-02-21 at 7 23 03 PM" src="https://github.com/calcom/cal.com/assets/53316345/5649def0-8554-4d2c-b50e-9c4410906e80">

Reply:
Plus https://cal.com/sales would display as text not clickable link

Reply:
ah yikes. can you fix this?

Reply:
great catch